duckduckgo/iOS
# cosmetic bug fixes

Reviewer: Mia
Asana: See below
CC:

**Description**:

Several small fixes based on feedback for the following tickets:

Update Major Tracker Network text to match Zeppelin:
https://app.asana.com/0/414235014887631/487161033888845

Apply correct text height/kerning to message in network leaderboard while gathering data:
https://app.asana.com/0/414235014887631/482056236481737

Increase the number of lines of error text and allow text to shrink to maximise space:
https://app.asana.com/0/414235014887631/487161033888836

Update privacy practices text:
https://app.asana.com/0/414235014887631/489057161850488

Use green for the b grade:
https://app.asana.com/0/414235014887631/488055746862776

**Steps to test this PR**:

1. Visit a site with trackers, open the network tracker detail, ensure correct message is displayed
1. Visit a site with major trackers, open the major tracker detail, ensure correct message is displayed
1. Reset network statistics, ensure gathering message text looks correct
1. Cause an error and ensure all text is visible (usually an SSL error will generate a longer message if possible, try searching for `all menus butchers daughter` and clicking through to almenus.com
1. Open privacy practices detail and ensure the text is correct
1. Visit sites with a B grade - ensure icon is green.
